Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM swiftdocker/swift:4.1
```

Comment:
Why do we need this Dockerfile? Can't we just directly use the swift:4.1 docker image?

Reply:
I added some steps in the dockerfile to pull the repo and branch that is being tested into the container volume

Reply:
I wish we did not need to have our own dockerfile for this, but it's a minor point, so I'll defer to your judgement.